postcss.config.js

```diff
@@ -1,37 +1,17 @@
 export default {
-    plugins: {
-        tailwindcss: {},
-        autoprefixer: {},
-        'postcss-pxtorem': {
-            rootValue: 16,
-            unitPrecision: 5,
-            propList: [
-                'font',
-                'font-size',
-                'line-height',
-                'letter-spacing',
-                'border*',
-                'background*',
-                'grid*',
-                'top',
-                'left',
-                'bottom',
-                'right',
-                'inset',
-                'width',
-                'height',
-                'margin*',
-                'padding*',
-                'max-*',
-                'min-*',
-                'gap*',
-            ],
-            selectorBlackList: [],
-            replace: true,
-            mediaQuery: true,
-            minPixelValue: 0,
-            exclude: /node_modules/i,
-        },
-        ...(process.env.NODE_ENV === 'production' ? { cssnano: {} } : {})
-    }
-}
\ No newline at end of file
+  plugins: {
+    tailwindcss: {},
+    autoprefixer: {},
+    'postcss-pxtorem': {
+      rootValue: 16,
+      unitPrecision: 5,
+      propList: ['*'],
```

Comment:
Works for me!

What do you think about putting a comment like this? Just to make it easy for folks to know they can exclude specific CSS properties if they'd like.

```
// Use ! to not match a property. Example: ['*', '!letter-spacing']
```

Reply:
and/or a comment linking to https://www.npmjs.com/package/postcss-pxtorem